docs/quickstart/installation-guide-for-beginners.mdx

```diff
@@ -213,66 +223,10 @@ This is the end of the one-time installations and the following steps will be re
 
       You should see the Jaeger UI. Keep this terminal window open to maintain access to Jaeger.
 
-    ##### 3. Configure Jaeger as a Destination in Odigos
-
-    a. Open a new terminal window.
-
-    b. Run the following command to open the Odigos UI:
-
-      ```
-      odigos ui
-      ```
-
-    c. In your web browser, a new tab will open with the Odigos UI (usually at http://localhost:3000).
-
-    d. In the Odigos UI, you'll see a "Select Sources" page. Click "Select All" to choose all available sources, then click "Next".
-
-    e. On the "Add Destinations" page, click on "Jaeger".
-
-    f. Fill in the following details:
-
-      - Destination Name: Enter "jaeger-demo" (or any name you prefer)
-      - Endpoint: Enter "jaeger.tracing"
-
-    g. Click the "Test Connection" button. If successful, you'll see a green checkmark.
-
-    h. Click "Create Destination".
```

Comment:
Should we describe somewhere how to setup the jaeger?

I see this below:

```
f. Fill in any required information. If you're not sure what to enter, consult the Odigos documentation or ask for help.
```

But for beginners it is better to make it explicit

Reply:
During my 1st and 2nd attempt at installing and settings things up, I didn't need any other info except for what is already described in the docs. I'm assuming anything else is optional?